app/Http/Controllers/TimelineController.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Message;
+
+class TimelineController extends Controller
+{
+    public function show()
+    {
+        $messages = Message::all([
```

Comment:
(suggest)
- ここのように明らかに将来的に問題になりそうな箇所は、TODOとしておきましょう。

```suggestion
        // TODO: offsetやlimitの指定が将来的に必要になりそう（ページングやレイジーロードがの検討）
        $messages = Message::all([
```